README.md

```diff
@@ -1 +1,184 @@
+</div>
+
+* This Website is a compaigne of music concert. 
+* It has the featured speakers. 
+* It has the footer showing the partners of the concert
+
+## 🛠 Built With <a name="built-with"></a>
+
+* Project was built with CSS, HTML and Javascript.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://redux.js.org/">Javascript</a></li>
+    <li><a href="https://redux.js.org/">HTML</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- *This website is highly responsive*
+- *The speakers section was rendered dynamically using javascript*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+LIVE DEMO
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://til2to.github.io/Capstone-I/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+<!-- In order to run this project you need: -->
+
+- *Computer with browser*
+- *Mobile device with browser*
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd your-local-folder
+- clone this repository into your local directory
+- open the project in your favorite code editor
+- Install node packages 
+- preview in your favorite browser
+
+
+### Install
+
+Install this project with: npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+### Deployment
+
+You can deploy this project using: 
+
+- Github pages or any host of your choice.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 *Author1*
+
+Dennis Owusu Prah
+
+- GitHub: [@til2to](https://github.com/til2to)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] *Add postgresql database*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project... thumbs up and please follow my github page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Hat tip to anyone whose code was used
+- Hat tip to anyone who inspired my codebase
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
```

Comment:
[OPTIONAL] This is a required changed but i have markd it as optional because the other code reviewers missed you are required to give acknowledgement to Cindy Shin like it says in the requirements.

The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Okay will do that right away